wallet_lib/wallet_factory.py

```diff
@@ -1,21 +1,29 @@
-from . import BCHWallet
-from . import BTCWallet
+from . import BCHWallet, BTCWallet
 from .wallet_exceptions import WalletException, WalletIsNotSupportedException
+from .adapters.wallet_adapter_base import WalletAdapterBase
+
 
 class WalletFactory:
 
-    _SUPPORTED_WALLETS = [ BCHWallet, BTCWallet ]
+    _SUPPORTED_WALLETS = {
+        'BCH': BCHWallet,
+        'BTC': BTCWallet
+    }
+
+    def __init__(self):
+        pass
 
-    def get(self, ticker_symbol):
+    def get(self, ticker_symbol, adapter: WalletAdapterBase):
```

Comment:
```suggestion
    def get(self, ticker_symbol, adapter: WalletAdapterBase = RPCAdapter(..args)):
```
Would be good to have a default strategy, like RPCAdapter. So the lib consumer should not always define it.
I propose to get RPC connection properties from environment variables.